2018/Furmanets/Championship/Championship/Tournamet.cs

```diff
@@ -3,120 +3,53 @@
-            {
-                meeting.FirstPlayer = _players[indexPlayer];
-                indexPlayer++;
-            }
-
-            foreach (var meeting in Standings.Tournament)
-            {
-                meeting.SecondPlayer = _players[indexPlayer];
-                indexPlayer++;
-                if (_players.Count == indexPlayer)
-                {
-                    break;
-                }
-            }
-
-            Draftsman.PaintTournamentStage(Standings);
+            IndexOfRound++;
         }
 
-        private void CollectorResults()
+        private void ChooseWinner(Meeting meeting)
         {
-            var nextRoundGrid = new TournamentGrid();
-            foreach (var meeting in Standings.Tournament)
-            {
-                ChoiseWinner(meeting, nextRoundGrid);
-            }
-
-            Console.Clear();
-            Draftsman.PaintTournamentRound(Standings);
-            Standings = nextRoundGrid;
-
-            if (Standings.Tournament.Count != 0)
-            {
-                Draftsman.PaintTournamentStage(Standings);
-                CollectorResults();
-            }
-        }
+            if (meeting.SecondPlayer == null || meeting.FirstPlayer == null)
+                return;
 
-        private void ChoiseWinner(Meeting meeting, TournamentGrid nextRoundGrid)
-        {
-            if (meeting.SecondPlayer != null)
+            if (meeting.Score[0] > meeting.Score[1])
             {
-                Draftsman.GetResultOfMatch(meeting);
-
-                if (meeting.Score[0] > meeting.Score[1])
-                {
-                    if (meeting.Stage == "final")
-                    {
-                        Draftsman.CongratulationWinner(meeting.FirstPlayer);
-                    }
-                    else
-                    {
-                        PromoteWinner(meeting, meeting.FirstPlayer, nextRoundGrid);
-                    }
-                }
-                else
-                {
-                    if (meeting.Stage == "final")
-                    {
-                        Draftsman.CongratulationWinner(meeting.SecondPlayer);
-                    }
-                    else
-                    {
-                        PromoteWinner(meeting, meeting.SecondPlayer, nextRoundGrid);
-                    }
-                }
+                PromoteWinner(meeting, meeting.FirstPlayer);
+                meeting.Winner = MeetingWinningIndicator.FirstPlayer;
             }
             else
             {
-                if (meeting.Stage == "final")
-                {
-                    Draftsman.CongratulationWinner(meeting.SecondPlayer);
-                }
-                else
-                {
-                    PromoteWinner(meeting, meeting.FirstPlayer, nextRoundGrid);
-                }
+                PromoteWinner(meeting, meeting.SecondPlayer);
+                meeting.Winner = MeetingWinningIndicator.SecondPlayer;
```

Comment:
If we have equal scores the method automaticly considers second player to be the winner of meeting